tests/foreman/ui/test_docker.py

```diff
@@ -1151,95 +1154,95 @@ class DockerContainersTestCase(UITestCase):
-        """@Test: Create a container in an external compute resource
-
-        @Feature: Docker
-
-        @Assert: The docker container is created in the external compute
-        resource
-
-        @Status: Manual
-
-        """
-
-    @stubbed()
-    @run_only_on('sat')
-    def test_create_container_local_compute_resource_power(self):
-        """@Test: Create a container in a local compute resource, then power it
-        on and finally power it off
-
-        @Feature: Docker
-
-        @Assert: The docker container is created in the local compute resource
-        and the power status is showing properly
-
-        @Status: Manual
-
-        """
-
-    @stubbed()
-    @run_only_on('sat')
-    def test_create_container_external_compute_resource_power(self):
-        """@Test: Create a container in an external compute resource, then
-        power it on and finally power it off
-
-        @Feature: Docker
-
-        @Assert: The docker container is created in the external compute
-        resource and the power status is showing properly
-
-        @Status: Manual
-
-        """
+        cls.organization = entities.Organization().create()
+        cls.lce = entities.LifecycleEnvironment(
+            organization=cls.organization).create()
+        cls.repo = entities.Repository(
+            content_type=u'docker',
+            docker_upstream_name=u'busybox',
+            product=entities.Product(organization=cls.organization).create(),
+            url=DOCKER_REGISTRY_HUB,
+        ).create()
+        cls.repo.sync()
+        content_view = entities.ContentView(
+            composite=False,
+            organization=cls.organization,
+        ).create()
+        content_view.repository = [cls.repo]
+        cls.content_view = content_view.update(['repository'])
+        cls.content_view.publish()
+        cls.cvv = content_view.read().version[0].read()
+        promote(cls.cvv, cls.lce.id)
+        cls.cr_internal = entities.DockerComputeResource(
+            name=gen_string('alpha'),
+            organization=[cls.organization],
+            url=settings.docker.get_unix_socket_url(),
+        ).create()
+        cls.cr_external = entities.DockerComputeResource(
+            name=gen_string('alpha'),
+            organization=[cls.organization],
+            url=settings.docker.external_url,
+        ).create()
+        cls.parameter_list = [
+            {'main_tab_name': 'Image', 'sub_tab_name': 'Content View',
+             'name': 'Lifecycle Environment', 'value': cls.lce.name},
+            {'main_tab_name': 'Image', 'sub_tab_name': 'Content View',
+             'name': 'Content View', 'value': cls.content_view.name},
+            {'main_tab_name': 'Image', 'sub_tab_name': 'Content View',
+             'name': 'Repository', 'value': cls.repo.name},
+            {'main_tab_name': 'Image', 'sub_tab_name': 'Content View',
+             'name': 'Tag', 'value': 'latest'},
```

Comment:
since we are changing from the python function function arguments to list of dictionaries, how do we make sure that the names like 'main_tab_name', 'sub_tab_name', etc passed from tests are indeed valid values?